File: problem/Min_K_Scores/minimumDifference.py
class Solution:
    def minimumDifference(self, nums, k):
        nums.sort()
        ans = nums[k - 1] - nums[0]
        for i in range(1, len(nums) - k + 1):
            ans = min(ans, nums[i + k - 1] - nums[i])
        return ans

File: problem/Min_K_Scores/minimumDifference.java
class Solution {
    public int minimumDifference(int[] nums, int k) {
        Arrays.sort(nums);
        int ans = nums[k - 1] - nums[0];
        for (int i = 1; i + k <= nums.length; i++) {
            ans = Math.min(ans, nums[i + k - 1] - nums[i]);
        }
        return ans;
    }
}